java/Makefile

```diff
@@ -30,8 +31,11 @@ else
   JAVA_RELEASE_NUMBER := $(shell echo $(JAVA_VERSION_STRING) | sed 's/\([0-9]*\)\..*/\1/')
 endif
 
+# Problem: Release 21-ea does not contain the "this-escape" lint flag.
```

Comment:
I would expect that we never use 21-ea, given that JDK 21 has been released.  Do you know where we use it?